app/experiments/faces_houses/params.ts

```diff
@@ -2,10 +2,13 @@ import * as path from 'path';
 import { EVENTS } from '../../constants/constants';
 
 // Default directories containing stimuli
-const rootFolder = __dirname; // Note: there's a weird issue where the fs readdir function reads from BrainWaves dir
+const resourcePath =
```

Comment:
@jdpigeon, just curious, is there a reason to redefine resourcePath here instead of using the one given in the constants.ts?